### DISCUSSION TITLE
Dashed ring on button

### DISCUSSION THREAD

**Discussion by Cromwell1963** [israel-dryer/ttkbootstrap#396] [Q&A]
Is it possible to disable the dashed ring on the Button?
If not, is it possible to select a button from the code?
Thanks

**Comment by israel-dryer** [comment]
Yes. The element you are referring to is called the focus ring and can be adjusted by setting the "focuscolor" to an empty string on the particular button style you are working with. Or you can simply use the "TButton" button style to affect all buttons.

```python
import ttkbootstrap as ttk

app = ttk.Window()

app.style.configure('TButton', focuscolor='')

ttk.Button(text='Button 1').pack(padx=5, pady=5)
ttk.Button(text='Button 2').pack(padx=5, pady=5)
ttk.Button(text='Button 3').pack(padx=5, pady=5)

app.mainloop()
```

<img width="90" alt="image" src="https://user-images.githubusercontent.com/42584742/209444193-479b6cdd-c194-46b4-b22d-73d364f573eb.png">

**Reply by rdbende** [reply]
Although I would like to add that disabling the focus indicator is generally bad practice, as users navigating with the keyboard cannot see which widget is selected.

**Reply by Cromwell1963** [reply]
That's really helpful, thank you

**Comment by Cromwell1963** [comment]
I am having problems making this work. Have i done it correctly?

```
    aetharcana = Main(
        title="Aetharcana",
        minsize=(1920, 1080),
        resizable=(True, True),
        hdpi=False,
        themename="aetharcana",
    )

    aetharcana.style.configure("TButton", focuscolor="")


```
Thanks
